src/Core/Framework/App/Manifest/Schema/manifest-3.0.xsd

```diff
@@ -491,6 +491,7 @@
     <xs:complexType name="gateways">
         <xs:sequence>
             <xs:element name="checkout" type="xs:anyURI" minOccurs="0"/>
+            <xs:element name="context" type="xs:anyURI" minOccurs="0"/>
```

Comment:
do we need to increase the manifest version? :thinking:

Reply:
@mitelg 
Afaik no, as these elements are all optional